environs/config/config.go

```diff
@@ -952,6 +972,14 @@ func (c *Config) MaxStatusHistorySizeMB() uint {
 	return uint(val)
 }
 
+// UpdateStatusHookInterval is how often to run the charm
+// update-status hook.
+func (c *Config) UpdateStatusHookInterval() time.Duration {
+	// Value has already been validated.
+	val, _ := time.ParseDuration(c.mustString(UpdateStatusHookInterval))
```

Comment:
My point is that when a user *types a value* they may just type 30 and the error message is going to be "not a string", which is a terrible user experience.

Given what we saw in the last round of "what happens when we introduce mandatory config", and the fact that IMO it is not ambiguous. An empty value can just be treated as "use the default", I'd really rather we layer defense in depth here so that we don't get bugs around it.